lib/amq/settings.rb

```diff
@@ -13,25 +13,35 @@ module Settings
       #
       # @see AMQ::Client::Settings.configure
       def self.default
-        @default ||= {
-          # server
-          :host  => "127.0.0.1",
-          :port  => AMQ::Protocol::DEFAULT_PORT,
+        # Consider to move this defaults to client libraries (bunny, amqp, auto-generated amq-protocol client)
```

Comment:
This comment should be deleted as these defaults won't move to Bunny and amqp gem.